### DISCUSSION TITLE
How to use the webdav server as the input source of liquidsoap

### DISCUSSION THREAD

**Discussion by dycfan** [savonet/liquidsoap#3014] [User Support]
**Thanks for using liquidsoap**
I use liquidsoap on ubuntu, and I need to use all audio files under a specified path on my webdav server as the input source of liquidsoap. I have tried many ways, without success. You can only ask for help here, hoping to get a response. Thank you so much! (my webdav server works fine; my liquidsoap version is 2.1.4)

**Comment by vitoyucepi** [comment]
Hi @dycfan,
There are several ways to solve this problem.
1. Get `davfs2` and mount the remote path.
1. Create a playlist and place it in webdav.
1. Write a script that will find desired files and compose a playlist.
1. Sync remote files to local fs.
1. Replace webdav with fs which can be mounted over the network.
1. Rewrite this issue as a feature request.

**Comment by dycfan** [comment]
Thank you so much for your guidance and help!

Mount the specific path of the webdav server locally as the input source of liquidsoap. I have used this method before, and the experience is not bad, but it is too cumbersome. I still hope that liquidsoap can support direct connection to the webdav server in the future, and use the specific path on the webdav server as the input source of liquidsoap.
